tests/test_module.rs

```diff
@@ -443,3 +443,21 @@ fn test_module_doc_hidden() {
         py_assert!(py, m, "m.__doc__ == ''");
     })
 }
+
+#[test]
+fn test_module_append_to_inittab() {
+    use pyo3::append_to_inittab;
+    append_to_inittab!(module_with_functions);
```

Comment:
You will probably find that this needs to go in its own test file, because it's quite likely that another test will already have initialised the Python interpreter.